### PR TITLE
Need Doc for Adding Certs to Registry

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-docs/issues/8375

### CODE REVIEW

**Comment by mburke5678** [comment]
@dmage Any help with this would be greatly appreciated.

**Comment by mburke5678** [comment]
@dmage BUMP

**Comment by dmage** [comment]
@mburke5678 I would like to have the exact commands how to get data for the `/etc/pki/tls/certs` directory, but I don't have any working RHEL system at this moment to write it. Otherwise it looks good.

**Comment by mburke5678** [comment]
@dmage @bparees 
Any advice on how to get data for the `/etc/pki/tls/certs` directory, as [Oleg suggested](https://github.com/openshift/openshift-docs/pull/8671#issuecomment-381152774)? Is  this what Ben was referring to in the original Issue PR  about [mounting the certs via a configmap or secret](https://github.com/openshift/openshift-docs/issues/8375#issue-307808808)? 
I did some searching around in our docs and online, but I have not found anything that clarifies this for me.

**Comment by mburke5678** [comment]
@dmage @bparees 
Any advice on how to get data for the `/etc/pki/tls/certs` directory, as [Oleg suggested](https://github.com/openshift/openshift-docs/pull/8671#issuecomment-381152774)? Is  this what Ben was referring to in the original Issue PR  about [mounting the certs via a configmap or secret](https://github.com/openshift/openshift-docs/issues/8375#issue-307808808)? 
I did some searching around in our docs and online, but I have not found anything that clarifies this for me.

**Comment by dongboyan77** [comment]
@mburke5678 LGTM, thanks

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: install_config/registry/extended_registry_configuration.adoc [hunk 1]
```diff
@@ -569,6 +569,14 @@ capability also makes it possible for you to pull images that reside on a
 registry they do not have credentials to access, as long as you have access
 to the image stream tag that references the image.
 
+You must ensure that your registry has appropriate certificatess to trust
```
Comment (kalexand-rh) [inline install_config/registry/extended_registry_configuration.adoc]: s/certificatess/certificates

File: install_config/registry/extended_registry_configuration.adoc [hunk 2]
```diff
@@ -569,6 +569,14 @@ capability also makes it possible for you to pull images that reside on a
 registry they do not have credentials to access, as long as you have access
 to the image stream tag that references the image.
 
+You must ensure that your registry has appropriate certificatess to trust 
+any external registries you do a pullthrough against. The certificates need to be placed 
+in the *_/etc/pki/tls/certs_* directory on the pod. You can mount the certifiates 
+using a xref:../..//dev_guide/configmaps.adoc#configmaps-creating-from-directories[configuration map]
```
Comment (kalexand-rh) [inline install_config/registry/extended_registry_configuration.adoc]: There's an extra / in this path.

File: install_config/registry/extended_registry_configuration.adoc [hunk 3]
```diff
@@ -569,6 +569,14 @@ capability also makes it possible for you to pull images that reside on a
 registry they do not have credentials to access, as long as you have access
 to the image stream tag that references the image.
 
+You must ensure that your registry has appropriate certificatess to trust 
+any external registries you do a pullthrough against. The certificates need to be placed 
+in the *_/etc/pki/tls/certs_* directory on the pod. You can mount the certifiates 
+using a xref:../..//dev_guide/configmaps.adoc#configmaps-creating-from-directories[configuration map] 
+or xref:../../dev_guide/secrets.adoc#dev-guide-secrets[secret]. Note that the entire the *_/etc/pki/tls/certs_* directory
```
Comment (kalexand-rh) [inline install_config/registry/extended_registry_configuration.adoc]: s/entire the/entire

File: install_config/registry/extended_registry_configuration.adoc [hunk 4]
```diff
@@ -569,6 +569,14 @@ capability also makes it possible for you to pull images that reside on a
 registry they do not have credentials to access, as long as you have access
 to the image stream tag that references the image.
 
+You must ensure that your registry has appropriate certificatess to trust 
+any external registries you do a pullthrough against. The certificates need to be placed 
+in the *_/etc/pki/tls/certs_* directory on the pod. You can mount the certifiates 
+using a xref:../..//dev_guide/configmaps.adoc#configmaps-creating-from-directories[configuration map] 
+or xref:../../dev_guide/secrets.adoc#dev-guide-secrets[secret]. Note that the entire the *_/etc/pki/tls/certs_* directory 
+must be replaced. You must include the new certificates and replace the system certificates 
+in your secret or configuration map that you are mounting.
```
Comment (kalexand-rh) [inline install_config/registry/extended_registry_configuration.adoc]: s/are mounting/mount

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks, but this sounds good. (I'd double-check your links locally.)

File: install_config/registry/extended_registry_configuration.adoc (hunk 2 above)
Comment (ahardin-rh) [inline install_config/registry/extended_registry_configuration.adoc]: extra / in the file path. Be sure to run asciibinder build to preview and test the links.

Comment (mburke5678) [reply install_config/registry/extended_registry_configuration.adoc]: Thanks. It did work in preview!

**Pull request merged**